actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -43,6 +43,18 @@ def initialize(params) # :nodoc:
     end
   end
 
+  # Raised when a Parameters instance is not marked as permitted and
+  # an operation to transform to hash is called.
+  #
+  #   params = ActionController::Parameters.new(a: "123", b: "456")
+  #   params.to_h
+  #   # => ActionController::UnfilteredParameters: unable to convert unpermitted parameters to hash
+  class UnfilteredParameters < StandardError
+    def initialize
+      super("unable to convert unpermitted parameters to hash")
```

Comment:
An exception don't need to explain what users have to do. It depends on the context and since we don't know what is the context we can't recommend something. But, it needs to be clear about what it means. Is the exception message and documentation clear about what it means?